Mathlib/MeasureTheory/Function/AbsolutelyContinuous.lean

```diff
@@ -375,4 +391,76 @@ theorem ae_differentiableAt {f : ℝ → ℝ} {a b : ℝ}
     ∀ᵐ (x : ℝ), x ∈ uIcc a b → DifferentiableAt ℝ f x :=
   hf.boundedVariationOn.ae_differentiableAt_of_mem_uIcc
 
+/-- If `f` is interval integrable on `a..b` and `c ∈ uIcc a b`, then `fun x ↦ ∫ v in c..x, f v` is
+absolute continuous on `uIcc a b`. -/
+theorem _root_.IntervalIntegrable.absolutelyContinuousOnInterval_intervalIntegral {f : ℝ → ℝ}
+    {a b c : ℝ} (h : IntervalIntegrable f volume a b) (hc : c ∈ uIcc a b) :
+    AbsolutelyContinuousOnInterval (fun x ↦ ∫ v in c..x, f v) a b := by
+  -- Step 1: Show that the function sending `E : ℕ × (ℕ → ℝ × ℝ)` to the total length of
```

Comment:
This first step does not depend on `f`, so it's probably worth factoring it out.